core/src/main/java/org/opensearch/sql/expression/aggregation/AggregatorFunction.java

```diff
@@ -65,7 +65,7 @@ private static DefaultFunctionResolver avg() {
         functionName,
         new ImmutableMap.Builder<FunctionSignature, FunctionBuilder>()
             .put(new FunctionSignature(functionName, Collections.singletonList(DOUBLE)),
-                arguments -> new AvgAggregator(arguments, DOUBLE))
+                (qc, arguments) -> new AvgAggregator(arguments, DOUBLE))
```

Comment:
Please rename these all to something more meaningful, like `queryContext` or `context`.  `qc` is confusing.

Reply:
Good point -- done.